examples/rating.py

```diff
@@ -0,0 +1,35 @@
+import easypost
+
+easypost.api_key = 'EASYPOSTAPIRATINGACTIVATEDKEY'
+
+rating = easypost.Rating.create(
+  carrier_accounts=["CARRIERACCOUNTID1", "CARRIERACCOUNTID2"],
+  to_address={
+    "name": 'Dr. Steve Brule',
```

Comment:
i know that the library is currently inconsistent about single vs double quotes. but i'm changing it to single quotes in my upcoming release. could you change all double quotes to single quotes here so that i can just copy this example in?